### ISSUE TITLE
Review Operate First website (https://www.operate-first.cloud/) and suggest ways to improve content organization

### ISSUE THREAD

**Issue by quaid** [operate-first/community#9]
This is a good task for everyone involved in the Marketing work to do at the start of their involvement. Having each person's un-influenced, honest experience and opinions is valuable at the start, so we are looking at the widest pool of understanding. (We don't want to pre-converge understanding, which can restrict the level and quality of ideas.)

**Comment by margarethaley** [comment]
# Thoughts / Initial Reactions to Operate First Site
I split this up by section, with notes on my reactions when reading that part of the website. I did not have something thoughtful to provide on every single sentence (particularly some of the more technical parts I'm unfamiliar with), so I tried to organize this so it's clear what part of the website I am referring to. Please let me know if anything is confusing!

## *[Home Page](https://www.operate-first.cloud/)*

*   “What is Operate First?”
    *   Could be simplified, more concise / direct - especially the first sentence
    *   Motivation behind Operate First is helpful to understanding the concept
*   “Why Operate First?”
    *   First time visiting the site, this quote really confused me because I had no idea what MOC was
    *   MOC is not explained on this page and it doesn’t link where you can find out that info - I had to go looking for it
*   Key Personas
    *   Developer/QE
        *   Points user to Operate First effort at the OI Labs community - but the link to OI labs community brings you to the MOC Introduction page
        *   Not immediately clear what the connection between MOC and OpenInfra Labs is, this could be confusing for the user
    *   Cloud Operator/SRE & Sys Admin
        *   Points user to the telemetry working group, links to an OI Labs page on telemetry
        *   No immediate info on how to join / get involved on this page
            *   Link to github at the very bottom
*   Benefits of Operate First
    *   Operationalize software: almost seems like more of a motivation than a benefit?
    *   Evaluate functionality and operability: sentence is a bit hard to follow, no oxford comma
*   Misc.
    *   Maybe could be made more visually appealing
    *   Less words so that info is not overwhelming
    *   Other pages for products at RH have a button near the top which users can click to get started right away, maybe something like that could be helpful

## *Data Science*

*   [Data Science Projects](https://www.operate-first.cloud/data-science/)
    *   Overview - proofreading
    *   Structure of the project pages is mostly (but not totally) consistent
        *   Ex: some projects have abstracts, others do not
*   [AI for Continuous Integration](https://www.operate-first.cloud/data-science/ocp-ci-analysis/)
    *   Clicked on overview first, which linked right to a page on how to get started. This had step by step instructions on how I could try it out right away myself, linked to other pages with more info

## *Users*

*   [Operate first support](https://www.operate-first.cloud/users/support/)
    *   Gives link to the home page, then says that it’s the support repo for Operate First - should this have been a link to the github page?
    *   Managed services
        *   Maybe it would be better to have a list of the services with a short description of what the service is / how it would benefit the user, and link to a page with more info on it
        *   The next page could provide info on how to get involved right away, and go more in depth on the service
    *   End user support, which links to the slack channel - should this be closer to the top of the page?
*   Mass Open Cloud
    *   [Introduction](https://www.operate-first.cloud/users/support/docs/intro.md)
        *   More of a generic getting started with Operate First page, which I wouldn’t have expected - I thought it would just be explaining the relationship with MOC
        *   Info on how to get started is super important, so maybe this could be linked from the home page?
        *   Some typos
*   General FAQ: there are no questions and answers in this section - just directions
*   Jupyterhub, ArgoCD, Prometheus, …, Kubeflow
    *   “How to access” inconsistencies
        *   Mostly all have this except for Jupyterhub
        *   ArgoCD’s is titled “Request access” instead - maybe better to have the consistency
    *   Maybe each one should have an intro/description of what it is and why you may want to use it

## *[Operations](https://www.operate-first.cloud/operations/)*

*   GitOps and Operate First
    *   Maybe define the GitOps model explicitly, and link to more info 

## *[Blueprints](https://www.operate-first.cloud/blueprints/blueprint/)*

*   Noticed some typos on the main page (ex: contains instead of containers ?, lightweight instead of lightweigh)

## Final Thoughts and Questions

*   “Operate First” - a noun and verb, used in different contexts that perhaps could be confusing
    *   Way to distinguish between Operate First as concept/approach, cloud environment, and action? 
*   What do people think of the language currently used on the site? Is it too technical, not technical enough? To what level should technical aspects be explained to the user, and how much technical background are we expecting from them?
*   Overall, I think proofreading/rewording could help make things more clear/concise and easier to digest - but maybe reworking the organization of the site is more of a priority?

**Comment by margarethaley** [comment]
# Community Audit Report Thoughts

Here are some of my reactions and comments to the recording of the Operate First audit report, which was derived from [this worksheet](https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html#_defining_healthy_communities) and [this generalized spreadsheet](https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html#_defining_healthy_communitie). Comments associated with a specific section of the report and video are marked with the section title and timestamp.

## Infrastructure (13:30)
*   Karsten mentioned renaming to Infrastructure of Participation, which I like - it seems more clear
*   Emphasized calls to actions, things you can get right away
*   Karsten discussed accessibility at the beginning - I think this could be crucial in encouraging open source collaboration
    *   The more accessible the site, the greater the chance at attracting users/collaborators
    *   Want to avoid someone visiting the site for the first time and feeling confused or excluded by the language, and then moving on
    *   Especially important with the homepage - the first things you see should be easily readable / understandable, grab your interest, and give you ways to get involved
    *   More technical should be accessible but perhaps not immediately seen, so as not to overwhelm some users

## Release Management (19:52)
*   Karsten said “rather than say it’s simple, show it’s simple” - absolutely!

## Documentation (27:20)
*   Documentation and UI changes have to match exactly
    *   If I am trying to follow directions, and they don’t match up with what I’m seeing, it can definitely cause confusion and frustration - want to avoid this
*   Tom’s point about the blurred line between Operate First and OpenInfra Labs
    *   As someone who’s knowledge of Operate First is based almost entirely on the website, I was not able to get a really clear understanding of the difference between the two and exactly how they interact
        *   Operate First is an initiative / approach / concept (kind of like upstream first), while OpenInfra Labs is a community where that initiative can be implemented?
*   Karsten mentions a diagram Bill created - I definitely think diagrams could be a huge help for explaining these things - visuals are often easier to comprehend and remember
*   Community handbook
    *   Could really help attract users/contributors to the initiative because it would provide explicit information and directions, make it easy to get involved
*   Joining meeting for MOC - potential barrier to entry or deterrent - it’s easy to miss something that happens once a month for an hour

## Contributions (38:21)
*   Really emphasizing organizational diversity to bring in more ideas, help move the initiative forward and keep the momentum going
*   Sentiment of wanting to watch out for others in the community is really great

## Outreach (45:30)
*   Code of conduct - important to make this easily accessible on the site
*   Blog and social media - we want to make it easy for people to interact with and contribute to these

## Overall Thoughts / Suggestions:

There was a discussion at the end of the recording on converting all tasks to issues on github, so it’s all in the open - this has been done and I think it’s great. Perhaps it could be beneficial to really organize these issues somehow. Consider their priority, and assign them to a priority level (if that’s possible within github?). Maybe identify any dependencies among them (for ex: developing the user personas is necessary to tailor proper language / messaging to those users). It also seems like many of these issues require more of an iterative process, rather than something that can be completed once and permanently checked off - maybe these issues can be distinguished and identified somehow in the github. I’m thinking this might help the team organize all the issues and determine the best way to proceed with them.
